support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -212,7 +252,7 @@ function WeeklyCheckoutForm(props: PropTypes) {
               </FormSection>
             : null
           }
-          <FormSection title="When would you like your subscription to start?">
+          <FormSection title={`When would you like ${props.orderIsAGift ? 'the' : 'your'} subscription to start?`}>
             <Rows>
               <FieldsetWithError id="startDate" error={firstError('startDate', props.formErrors)} legend="When would you like your subscription to start?">
```

Comment:
Should we use the same adjusted copy as above here? In which case maybe extract it to a const first

Reply:
👍